build.sbt

```diff
@@ -2,19 +2,18 @@
  * Copyright (C) 2015-2019 Lightbend Inc. <https://www.lightbend.com>
  */
 
- import interplay.ScalaVersions._
+import interplay.ScalaVersions._
 
-// To keep all the Scala versions consistent with the Scala versions
-// provided by interplay.
- val commonSettings = Seq(
-   scalaVersion := scala212,
-   crossScalaVersions := Seq(scala211, scala212)
- )
+val commonSettings = Seq(
+  scalaVersion := scala212,
+  crossScalaVersions := Seq(scala212)
```

Comment:
Can't support Scala 2.11, because hasn't compatible version of `"com.typesafe.play" % "sbt-plugin"`.
But maybe I missing something. 😞